test_communication/CMakeLists.txt

```diff
@@ -179,13 +179,17 @@ if(BUILD_TESTING)
       set(SKIP_TEST "SKIP_TEST")
     endif()
 
-    # TODO(mikaelarguedas) connext doesn't support C services for now
-    if((client_library1 STREQUAL "rclpy" OR client_library2 STREQUAL "rclpy") AND
-      (rmw_implementation1 MATCHES "(.*)connext(.*)" OR rmw_implementation2 MATCHES "(.*)connext(.*)")
+    # TODO(mikaelarguedas) connext_dynamic doesn't support C services for now
+    if((client_library1 STREQUAL "rclpy" AND rmw_implementation1 MATCHES "rmw_connext_dynamic_cpp") OR
+      (client_library2 STREQUAL "rclpy" AND rmw_implementation2 MATCHES "rmw_connext_dynamic_cpp")
     )
       set(SKIP_TEST "SKIP_TEST")
     endif()
 
+    if(WIN32 AND client_library1 STREQUAL "rclcpp" AND client_library2 STREQUAL "rclpy")
```

Comment:
Why is this case skipped? I comment would help to clarify.

Reply:
Looks like this branch is missing https://github.com/ros2/system_tests/pull/248. These tests should not be skipped: https://github.com/ros2/system_tests/pull/249/commits/13a4feb7fc7a942e0fd906289997affc411b92a4

Reply:
Note that the Windows job had both changes so these tests were indeed ran on CI (e.g. http://ci.ros2.org/job/ci_windows/3866/testReport/test_communication.build.test_communication/test_requester_replier__Empty__rclcpp__rclpy__rmw_connext_cpp_Release/)